### ISSUE TITLE
"serverBundle: auto" seems not work on cloudflare

### ISSUE THREAD

**Issue by HADB** [nuxt/icon#213]
I'm using [Cloudflare Pages GitHub/GitLab integration](https://developers.cloudflare.com/pages/get-started/#connect-your-git-provider-to-pages), and the log shows "[info] Nuxt Icon server bundle mode is set to `local`" when building which causes the bundle size exceed the 25M limit. It should be `remote` as the document saying.

It works fine if I set `serverBundle` to `remote` manually.

**Comment by antfu** [comment]
Could you help checking how we could improve the detection logic here? I don't use CF a lot to know all the gotchas.
https://github.com/nuxt/icon/blob/ec956151c0e195b12be7f57fffe38c584de43306/src/module.ts#L66-L70

**Comment by HADB** [comment]
OK, I will try today.

> Could you help checking how we could improve the detection logic here? I don't use CF a lot to know all the gotchas.
> 
> https://github.com/nuxt/icon/blob/ec956151c0e195b12be7f57fffe38c584de43306/src/module.ts#L66-L70

**Comment by HADB** [comment]
@antfu Hi, I figured out the problem and created a PR here, please have a look.

The problem is that nuxt.options.nitro.preset was undefined in my cloudflare build environment with zero-config support of nitro, but there's a environment variables NITRO_PRESET:"cloudflare_pages" we can use to detect the preset.

**Comment by MickL** [comment]
Is this actually fixed? I am on latest `1.4.5` and my bundle size just went up from 2MB to more than 25 MB and deployments now fail.

```
[info] Nuxt Icon server bundle mode is set to `local`
[success] Nuxt Icon discovered local-installed 9 collections: bi, carbon, heroicons, hugeicons, logos, material-symbols, ri, solar, tabler
```

Btw. From this icons I only use 3 (bi, hugeicons and solar), I dont know why it says 9.

**Comment by MickL** [comment]
I tried. Seems to work with 1.4.6 that was released yesterday.
